security/form_login.rst

```diff
@@ -74,7 +74,7 @@ First, enable ``form_login`` under your firewall:
 
 Now, when the security system initiates the authentication process, it will
 redirect the user to the login form ``/login``. Implementing this login form
-is your job. First, create a new ``SecurityController`` inside a bundle::
+is your job. First, create a new ``SecurityController`` inside your app::
```

Comment:
Thank you, good catch!

Could you please change “app” to “application”?

Reply:
We may even remove the entire "inside your app" ... because previously it made sense, but now it looks redundant.